python/lsst/pipe/tasks/ingestCalibs.py

```diff
@@ -0,0 +1,164 @@
+import collections
+import datetime
+import sqlite3
+from lsst.pex.config import Config, ListField, ConfigurableField
+from lsst.pipe.base import Task, ArgumentParser
+from lsst.pipe.tasks.ingest import RegisterTask, ParseTask, RegisterConfig
+
+def _convertToDate(dateString):
+    """Convert a string into a date"""
+    try:
+        return datetime.datetime.strptime(dateString, "%Y-%m-%d").date()
+    except ValueError:
+        return None
+
+class CalibsParseTask(ParseTask):
+    """Task that will parse the filename and/or its contents to get the
+    required information to populate the calibration registry."""
+    def translate_obstype(self, md):
+        """Convert the observation type in the header
+        to a known calibration dataset type.
+
+        @param md: FITS header metadata
+        """
+        obstype = md.get("OBSTYPE").strip().lower()
+        if "flat" in obstype:
+            obstype = "flat"
+        elif "zero" in obstype or "bias" in obstype:
+            obstype = "bias"
+        elif "dark" in obstype:
+            obstype = "dark"
+        elif "fringe" in obstype:
+            obstype = "fringe"
```

Comment:
I'd say no.  That file would just be ignored and not written into the calibration registry.